src/NumberGuessingGame.jsx

```diff
@@ -1,77 +1,52 @@
-import React, { Component } from "react";
-import GuessControl from "./GuessControl";
-import GuessMessage from "./GuessMessage";
-import GameOver from "./GameOver";
+import React, { Component } from 'react';
```

Comment:
You need to import the useState property from 'react' to make use in lines 13, 14 and 15.